### ISSUE TITLE
Seeding Input: filter area dropdown for tray vs direct seeding

### ISSUE THREAD

**Issue by IrisSC** [DickinsonCollege/FarmData2#266]
The area dropdown should only show greenhouses when Tray seeding is selected and it should only show fields or beds when Direct seeding is selected. This should be dynamic, the user should be able to switch back and forth. If the user selects a Tray seeing and a greenhouse, but then selects direct seeding the area should be switched to blank and vice versa. The type of area is located in the information obtained in api call using this link under “area_type”: http:/localhost/taxonomy_term.json?bundle=farm_areas

To test this go to the Seedings Input subtab under FieldKit tab, which is on the home page.  

The code can be found by going to farmdata2_modules/fd2_tabs/fd2_field_kit/seedingInput.html file.

**Comment by IrisSC** [comment]
I've started working on this issue. It looks like, there are three different types of areas: field, bed, and greenhouse. Based on what I have seen it looks like the beds are in the greenhouse, but direct seeded, so I am going to code them as showing up when direct seeding is selected. I want to make sure that this is correct.

**Comment by braughtg** [comment]
> looks like the beds are in the greenhouse, but direct seeded, so I am going to code them as showing up when direct seeding is selected. I want to make sure that this is correct.

That is correct.  The beds in the greenhouse are direct seeded in the ground in the rows down the center of the greenhouse.  The tray seedings occur on the benches on the sides.
